docs/windowsbrush-and-theme.md

```diff
@@ -15,48 +15,69 @@ In this example, we'll look at three things:
 - How to switch styles when a theme change has occurred
 - Handling a theme changed event
 
-#### Setting up your app to be sensitive to theme changes
+#### Using hooks to be sensitive to theme changes
 
-First import the Platform API into your React Native app.
+First import the `useColorScheme` hook into your React Native app.

```

Comment:
Consider adding a warning (not sure if we have a warning box) that this won't work over web debugging. It's a limitation of core, but it has consistently tripped people up.

Reply:
Oh whoops. Wrong section...

Reply:
There is a warning box a few lines down.